docs/tutorials/quickstart-mdldev.rst

```diff
@@ -31,12 +31,17 @@ information about each, visit the :ref:`Experiment Configuration Reference
 Software
 ========
 
--  Determined *agent* and *master* nodes must be configured with Ubuntu 16.04 or higher, CentOS 7,
-   or macOS 10.13 or higher.
+-  Determined *agent* and *master* nodes must be configured with the following operating systems:
+
+   - Ubuntu 20.04 or higher
+   - Enterprise Linux (e.g., AlmaLinux, Oracle Linux, Red Hat Enterprise Linux, or Rocky Linux) 7 or higher
+   - Windows Subsystem for Linux (WSL) 2 on Windows 10 version 1903 or higher, with a supported Linux distribution
+     installed
+   - macOS 10.13 or higher
 
 -  Agent nodes must have Docker installed.
 
--  To run jobs with GPUs, install NVIDIA drivers, version 384.81 or higher, on each agent. The
+-  To run jobs with GPUs, NVIDIA drivers, version 384.81 or higher, are required on each agent. The
```

Comment:
To run jobs that use GPUs, you need NVIDIA drivers with a version of 384.81 or later installed on each agent. These drivers can be installed independently, but you don't need to install the entire CUDA toolkit.